elide-standalone/src/test/java/example/ElideStandaloneDisableGraphQLTest.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2018, Oath Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example;
+
+import static com.yahoo.elide.Elide.JSONAPI_CONTENT_TYPE;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attr;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attributes;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.data;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.id;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.resource;
+import static com.yahoo.elide.test.jsonapi.JsonApiDSL.type;
+import static io.restassured.RestAssured.given;
+//import static org.hamcrest.CoreMatchers.equalTo;
+import com.yahoo.elide.ElideSettings;
+import com.yahoo.elide.ElideSettingsBuilder;
+import com.yahoo.elide.core.datastore.DataStore;
+import com.yahoo.elide.core.dictionary.EntityDictionary;
+import com.yahoo.elide.core.filter.dialect.RSQLFilterDialect;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.SQLDialectFactory;
+import com.yahoo.elide.jsonapi.links.DefaultJSONApiLinks;
+import com.yahoo.elide.standalone.ElideStandalone;
+import com.yahoo.elide.standalone.config.ElideStandaloneAnalyticSettings;
+import com.yahoo.elide.standalone.config.ElideStandaloneAsyncSettings;
+import com.yahoo.elide.standalone.config.ElideStandaloneSettings;
+import example.models.Post;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.util.Properties;
+import java.util.TimeZone;
+
+/**
+ * Tests ElideStandalone starts and works.
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ElideStandaloneDisableGraphQLTest {
+
+    protected ElideStandalone elide;
+
+    @BeforeAll
+    public void init() throws Exception {
+        elide = new ElideStandalone(new ElideStandaloneSettings() {
+
+            @Override
+            public ElideSettings getElideSettings(EntityDictionary dictionary, DataStore dataStore) {
```

Comment:
Can we minimize all of this configuration to as little as possible to simplify the test?